Source/WebKit/GPUProcess/GPUConnectionToWebProcess.cpp

```diff
@@ -190,10 +190,12 @@ class GPUProxyForCapture final : public UserMediaCaptureManagerProxy::Connection
     }
 
 private:
-    Logger& logger() final { return m_process.get()->logger(); }
+    Logger& logger() const final { return m_process.get()->logger(); }
```

Comment:
Why are you making these functions const?

Reply:
I was adding a `protectedLogger()` function right next to `logger()` and saw that `logger()` doesn't modify anything and could be marked const--so I thought I'd add it. Also I see that many `protectedFoo()` functions are const so I thought I'd made this one const if possible, which meant making `logger()` const as well.

Reply:
Actually this const is wrong because calling logger does modify the `m_process` member variable. I'll remove this const.

Reply:
Fixed!